webviz_subsurface/plugins/_prod_misfit/_plugin.py

```diff
@@ -255,6 +253,23 @@ def _get_wells_vectors_phases(
     return wells, vectors, list(phases)
 
 
+def _skip_well(
+    well: str,
+    excl_name_startswith: list,
+    excl_name_contains: list,
+) -> bool:
+    """Check well name against exclude strings and return True if it should be skipped."""
+
+    # ensure all list elements are str
+    excl_name_startswith = [str(element) for element in excl_name_startswith]
```

Comment:
```suggestion
    excl_name_startswith = [str(element) for element in excl_name_startswith]
```
What other types do we expect than `str`? Can we clean this "further up" and ensure `excl_name_startswith = List[str]` to begin with?

Reply:
int could be expected, though str would be more common. In the case of int input, the user can now use the list form [5, 36] as alternative to ["5", "36"]